### PR TITLE
Tests setup: tox setup, coveragerc, test requirements, flake8 issues.

### PR DESCRIPTION
**Pull request by cyberstar**
Initial tests infrastructure setup.

* test requirements,
* test settings,
* tox.ini
* setup changes
* minor flake8 issues.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+language: python
+python: 2.7
+env:
+  - TOX_ENV=flake8
+  - TOX_ENV=py27-dj16-cms30
+  - TOX_ENV=py27-dj16-cms31
+  - TOX_ENV=py27-dj17-cms30
+  - TOX_ENV=py27-dj17-cms31
```
Comment (mkoistinen) [inline .travis.yml]: Let's put the latest versions of things on top, so, should have dj17 tests before dj16 tests and cms31 tests before cms30 tests, etc.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+matrix:
+  include:
+    - python: 2.6
```
Comment (mkoistinen) [inline .travis.yml]: No need to declare the interpreter here. Tox handles that.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+language: python
+python: 2.7
```
Comment (mkoistinen) [inline .travis.yml]: No need to declare the interpreter here. Tox handles that.

File: aldryn_reversion/utils.py [hunk 4]
```diff
@@ -27,7 +27,7 @@ def build_obj_repr(obj):
     Returns a unicode string of object Model name and its text representation.
     """
     return "{0}: '{1}'".format(force_text(obj._meta.model.__name__),
-                                force_text(obj))
+                               force_text(obj))
```
Comment (mkoistinen) [inline aldryn_reversion/utils.py]: I notice that you're only loading force_text(), what about older Django's who only have force_unicode()?

File: tox.ini [hunk 5]
```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = flake8, {py26,py27,py33,py34}-dj{16,17}-cms{30,31}
```
Comment (mkoistinen) [inline tox.ini]: Please use:

`flake8, py{34,27}-dj{18,17}-cms{31,30}, {py27,py26}-dj16-cms{31,30}`

This avoids configurations that won't work with one another.

Then, you can create your envlist for use in .travis.yml with:

````
tox -l
````

Which yields:

````
py34-dj18-cms31
py34-dj18-cms30
py34-dj17-cms31
py34-dj17-cms30
py27-dj18-cms31
py27-dj18-cms30
py27-dj17-cms31
py27-dj17-cms30
py27-dj16-cms31
py27-dj16-cms30
py26-dj16-cms31
py26-dj16-cms30
````

File: aldryn_reversion/utils.py (hunk 4 above)
Comment (cyberstar) [inline aldryn_reversion/utils.py]: Actually I'm a little bit confused with those forces, https://github.com/django/django/blob/0cbe3c63af72233877e95ef24fe571cecc7cad9f/django/utils/encoding.py#L163
What is the best practice for forcing text repr across misc django versions? Also it seems that those force_unicode = force_text is true for django versions 1.6, 1.7 and 1.8...

Comment (mkoistinen) [inline aldryn_reversion/utils.py]: I usually do:

````
try:
    from django.utils.translation import force_text
except:
    from django.utils.translation import force_unicode as force_text
````

Comment (mkoistinen) [inline aldryn_reversion/utils.py]: But perhaps this is a hold over from pre-Django 1.6 days. Maybe we're cool with what you're doing then =)

File: tox.ini (hunk 5 above)
Comment (cyberstar) [inline tox.ini]: Thanks! Didn't knew this feature.

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**